src/Messages.js

```diff
@@ -1194,8 +1194,8 @@ export default defineMessages({
     description: 'No Pathways available',
     defaultMessage: 'No pathways available',
   },
-  pathwaysPanelsError: {
-    id: 'pathwaysPanelError',
```

Comment:
Hm.... `pathwaysPanelsError` is being removed, but it is still used in [OverviewDashbar.js](https://github.com/RedHatInsights/insights-advisor-frontend/pull/1164/files#diff-8938054b4ab4eaabf61d8d03883a36e7881a5873184cf696ce8eea6183ca0ceb)

Reply:
`pathwaysPanelsError` use was changed to the intended replacement `overviewDashbarError`
Thank you for noticing 🙏🏼 😅

Reply:
The use cases of `pathwaysPanelsError` found in data.json and translations.json were also removed, because there was no other use case anywhere in the application